docs/intro/tutorial08.txt

```diff
@@ -0,0 +1,95 @@
+=====================================
+Writing your first Django app, part 8
+=====================================
+
+This tutorial begins where :doc:`Tutorial 7 </intro/tutorial07>` left off. We've
+built our web-poll application and will now look at third-party packages. One of
+Django's strengths is the rich ecosystem of third-party packages. They're
+community developed packages that can be used to quickly improve the feature-set
+of an application.
+
+This tutorial will show how to add `Django Debug Toolbar
+<https://django-debug-toolbar.readthedocs.io>`_, a commonly used third-party
+package. The Django Debug Toolbar has ranked in the top three most used
+third-party packages in the Django Developers Survey in recent years.
+
+.. admonition:: Where to get help:
+
+    If you're having trouble going through this tutorial, please head over to
+    the :doc:`Getting Help</faq/help>` section of the FAQ.
+
+Installing Django Debug Toolbar
+===============================
+
+Django Debug Toolbar is a useful tool for debugging Django web applications.
+It's a third-party package maintained by the `Jazzband
+<https://jazzband.co>`_ organization. The toolbar helps you understand how your
+application functions and to identify problems. It does so by providing panels
+that provide debug information about the current request and response.
+
+To install a third-party application like the toolbar, you need to install
+the package by running the below command within an activated virtual
+environment. This is similar to our earlier step to :ref:`install Django
+<installing-official-release>`.
+
+.. console::
+
+    $ python -m pip install django-debug-toolbar
+
+Third-party packages that integrate with Django need some post-installation
+setup to integrate them with your project. Often you will need to add the
+package's Django app to your :setting:`INSTALLED_APPS` setting. Some packages
+need other changes, like additions to your URLconf (``urls.py``).
+
+Django Debug Toolbar requires several setup steps. Follow them in `its
+installation guide
+<https://django-debug-toolbar.readthedocs.io/en/latest/installation.html>`_.
+The steps are not duplicated in this tutorial, because as a third-party
+package, it may change separately to Django's schedule.
+
+Once installed, you should be able to see the DjDT "handle" on the right side
+of the browser window when you refresh the polls application.
```

Comment:
Currently the tutorial basically ends here - install and see DjDT. We should say "click to open" otherwise it doesn't seem like the toolbar does much 😅

```suggestion
Once installed, you should be able to see the DjDT "handle" on the right side
of the browser window when you refresh the polls application. Click it to
open the debug toolbar and use the tools in each panel. See the `panels
documentation page
<https://django-debug-toolbar.readthedocs.io/en/latest/panels.html>`__ for
more information on what the panels show.
```